deployment/troposphere/tiler_hosted_zone_template.py

```diff
@@ -0,0 +1,77 @@
+from troposphere import Template, Parameter, Ref, GetAtt
+
+import template_utils as utils
+import troposphere.route53 as r53
+import troposphere.cloudfront as cf
+
+t = Template()
+
+t.add_version('2010-09-09')
+t.add_description('Tiler hosted zone records for the nyc-trees project.')
+
+#
+# Parameters
+#
+hosted_zone_name_param = t.add_parameter(Parameter(
+    'PublicHostedZone', Type='String',
+    Default='treescount.azavea.com',
+    Description='Hosted zone name for public DNS'
+))
+
+tile_server_hosted_zone_alias_target_param = t.add_parameter(Parameter(
+    'TileServerAliasTarget', Type='String',
+    Description='Alias target for the hosted zone record set'
+))
+
+#
+# CloudFront Resources
+#
+cloudfront_tile_distribution = t.add_resource(cf.Distribution(
+    'tileDistribution',
+    DistributionConfig=cf.DistributionConfig(
+        Origins=[
+            cf.Origin(
+                Id='tileOriginId',
+                DomainName=Ref(tile_server_hosted_zone_alias_target_param),
+                CustomOriginConfig=cf.CustomOrigin(
+                    OriginProtocolPolicy='http-only'
```

Comment:
This setting asks CloudFront to speak to the origin (the tiler ELB) with only HTTP.